lib/Viroverse/Controller/Sequence.pm

```diff
@@ -77,6 +77,22 @@ sub show : Chained('load') PathPart('') Args(0) {
     $c->detach( $c->view("NG") );
 }
 
+sub delete : Chained('load') PathPart('') Args(0) DELETE {
+    my ($self, $c) = @_;
+
+    my $seq = $c->stash->{current_model_instance};
```

Comment:
The pattern with this is setting `current_model_instance` in the stash in the action being chained off makes the instance available as `$c->model` in the next action, which is the preferred way to access it, there's examples elsewhere.